config/nginx-default.conf

```diff
@@ -0,0 +1,28 @@
+server {
+    server_name localhost;
+    listen 80;
+    root /srv/annotations/web;
+
+    location / {
+        try_files $uri /app_dev.php$is_args$args;
+    }
+
+    location ~ ^/app_dev\.php(/|$) {
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        include fastcgi_params;
+        fastcgi_param SCRIPT_FILENAME $request_filename;
+        fastcgi_param ENVIRONMENT_NAME dev;
+        fastcgi_intercept_errors on;
+        fastcgi_pass php:9000;
+        internal;
+    }
+
+    # return 404 for all other php files not matching the front controller
+    # this prevents access to other php files you don't want to be accessible.
+    location ~ \.php$ {
+        return 404;
+    }
```

Comment:
have to find a way to reduce the boilerplate of this file, but it's early to say. An orchestrator like Kubernetes would have its own way to create configuration files and mount them in container, but we are just talking about local usage here.